app/src/main/res/layout/user_grid.xml

```diff
@@ -0,0 +1,80 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.cardview.widget.CardView xmlns:android="http://schemas.android.com/apk/res/android"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    android:layout_marginEnd="@dimen/margin_6dp"
+    android:layout_marginBottom="@dimen/margin_12dp"
+    android:layout_marginStart="@dimen/margin_6dp"
+    android:layout_marginTop="@dimen/margin_6dp"
+    app:cardCornerRadius="@dimen/radius_10dp"
+    app:cardElevation="@dimen/elevation_8dp"
+    android:padding="@dimen/margin_12dp"
+    android:backgroundTint="@color/background_grey">
+
+    <androidx.constraintlayout.widget.ConstraintLayout
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
+        android:padding="@dimen/margin_12dp">
+
+        <de.hdodenhof.circleimageview.CircleImageView
+            android:id="@+id/profile_image"
+            android:layout_width="@dimen/img_100dp"
+            android:layout_height="@dimen/img_100dp"
+            android:src="@drawable/ic_launcher_foreground"
+            app:civ_border_width="@dimen/img_border_2dp"
+            app:civ_border_color="@color/border_dark"
+            app:layout_constraintEnd_toEndOf="parent"
+            app:layout_constraintTop_toTopOf="parent"
+            app:layout_constraintStart_toStartOf="parent"/>
+
+        <TextView
+            android:id="@+id/profile_name"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_marginTop="@dimen/margin_12dp"
+            android:fontFamily="sans-serif-medium"
+            android:text="@string/john_doe"
+            android:textColor="@color/grey_text"
+            android:textSize="@dimen/profileName_30sp"
+            android:textStyle="bold"
+            app:layout_constraintEnd_toEndOf="parent"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintTop_toBottomOf="@id/profile_image" />
+
+        <LinearLayout
+            android:layout_width="wrap_content"
```

Comment:
use constraint layout

Reply:
This was for my buttons. It seemed like less code with linear layout. But i will use constraint layout if you prefer.